README.md

```diff
@@ -146,42 +146,31 @@ If it's not `healthy` you should test your logstash configuration.
-You can use the special key 'host-path' to configure file-plugins inside logstash input-blocks to indicate that a file should be logged from the slave itself (instead of the executor's docker container).
-Example:
-```
-input { file { "host-path" => "/var/log/hello.log" } }
-```
-This will configure logstash to monitor the file `/var/log/hello.log` on all slaves. Note: this requires that the logstash framework is configured to use `/var/log` as a volume.
-
-### Docker Configuration
-Docker configurations will be propagated to slaves running docker containers with image names matching the docker configuration. Note that tags are usually included in docker image names.
-
-Example: a slave running a docker container with the nginx:latest docker image will be provided a docker configuration with 'applicable image name' set to nginx:latest, if it exists.
-
-You can use the special key 'docker-path' to configure file-plugins inside logstash input-blocks to indicate that a file should be logged from within the docker container itself.
-
-Example:
-```
-input { file { "docker-path" => "/var/log/hello.log" } }
-```
-
-This will configure logstash to monitor the file `/var/log/hello.log` inside all applicable docker containers.
+### Configuration
+
+The Logstash framework is configured at the time that the scheduler is started. Each configuration option can be passed in a large number of ways:
+
+1. Command-line arguments, e.g. `java -jar logstash-mesos-scheduler.jar --logstash.heapSize=64`
+2. Environment variables, e.g. `LOGSTASH_HEAP_SIZE=64 java -jar logstash-mesos-scheduler.jar`
+3. A properties file, e.g. `echo 'logstash.heap-size=64' > ./application.properties && java -jar logstash-mesos-scheduler.jar` 
+4. ...
+
+| Command-line argument            | Environment variable           | What it does                                                                                                               |
+| -------------------------------- | ------------------------------ | -------------------------------------------------------------------------------------------------------------------------- |
+| `--zk-url U`                     | `ZK_URL=U`                     | The Logstash framework will find Mesos using ZooKeeper at URL `U`, which must be in the format `zk://host:port/zkNode,...` |
+| `--zk-timeout T`                 | `ZK_TIMEOUT=T`                 | The Logstash framework will wait `T` milliseconds for ZooKeeper to respond before assuming that the session has timed out  |
+| `--framework-name N`             | `FRAMEWORK_NAME=N`             | The Logstash framework will show up in the Mesos Web UI with name `N`, and the ZK state will be rooted at znode `N`        |
+| `--webserver-port P`             | `WEBSERVER_PORT=P`             | The scheduler will listen on TCP port `P` FIXME and host what on it? Not clear from code                                   |
```

Comment:
@mwl Do you know what this option actually does? Afaics it just does `new JettyEmbeddedServletContainerFactory(P)` and then discards the factory